android/src/main/java/com/thebluealliance/androidclient/accounts/AccountHelper.java

```diff
@@ -96,8 +97,17 @@ public static GoogleAccountCredential getSelectedAccountCredential(Context conte
         return credential;
     }
 
-    public static TbaMobile getTbaMobile(GoogleAccountCredential credential) {
+    public static TbaMobile getTbaMobile(Context context, GoogleAccountCredential credential) {
         TbaMobile.Builder tbaMobile = new TbaMobile.Builder(HTTP_TRANSPORT, JSON_FACTORY, credential);
+        if (Utilities.isDebuggable()) {
+            // If a debug build, set Cloud Endpoints to point to a dev TBA instance
+            String appspotId = Utilities.readLocalProperty(context, "appspot.projectId");
+            if (!appspotId.isEmpty()) {
```

Comment:
It defaults to `tbatv-prod-hrd` (main site) if nothing is there. Which won't work because the keys aren't set up, but it's best to have these fail with auth errors, rather than 404.

This is consistent with how it works now